lib/linux_admin/process/memory.rb

```diff
@@ -0,0 +1,49 @@
+module LinuxAdmin
+  module Process
+    class Memory
+      attr_reader :pss, :rss, :uss, :shared, :swap, :size
```

Comment:
Can you give meaningful-named aliases to pss, rss, uss...I'll never remember them as they are and LinuxAdmin aims for clarity over these esoteric things :)